cpp/ql/src/Likely Bugs/Protocols/TlsSettingsMisconfiguration.qhelp

```diff
@@ -6,6 +6,17 @@
   <p>Using TLS or SSLv23 protool from the boost::asio library, but not disabling deprecated protocols or disabling minimum-recommended protocols.</p>
```

Comment:
Same comments on this as for:
https://github.com/Semmle/ql/pull/2113/files#diff-5bc34902af26176dba7dcfb06071fad3
fixed in https://github.com/Semmle/ql/pull/2372/files#diff-bf6ab0fb3282084837b92e3c632edb11

Reply:
Those parts of the file aren't changed by this PR, so the fixes in the older PR will remain when this one is merged.  That said, I've just rebased this PR onto a more recent master so that you can see the end result here.